build.gradle

```diff
@@ -19,6 +19,7 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.junit.jupiter:junit-jupiter:5.8.1'
```

Comment:
Tahle zavislost mi v kontextu ukolu uplne nedava smysl. Je to priprava na dalsi ukoly?